rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -749,15 +751,21 @@ public URI enlistCompensator(URI uri, Long timelimit, String linkHeader, StringB
         try {
             client = getClient();
             try {
+
+                MultivaluedMap<String, Object> headers = new MultivaluedHashMap<>();
+                headers.add(NARAYANA_LRA_API_VERSION_HEADER_NAME, LRAConstants.CURRENT_API_VERSION_STRING);
+                if (!data.isEmpty()) {
```

Comment:
OK but it's not as readable. In that case I will invoke our rule that says changes must conform to the extant style in the file being changed.

But I will say that you can't use the fact that you haven't configured your IDE correctly to justify adding code that breaks our style guidelines.

Reply:
Thanks Mike, I am working to configure my Eclipse IDE correctly in order to have a list of  'Save actions' coherent with the code. This list of actions applies to the edited lines only. When it will be completed I can share my preference file (.epf format) to any contributors that might want it. I think it would be great to have a coherent style and format in all the jbosstm repos in the future. Thanks for the patience.